src/feedback/AlertList.jsx

```diff
@@ -11,7 +11,7 @@ class AlertList extends Component {
   getUserMessage(code, userMessage) {
     if (this.props.messageCodes[code]) {
       return this.props.messageCodes[code];
-    } else if (code === 'fallback-error') {
+    } if (code === 'fallback-error') {
```

Comment:
The linter doesn't complain about having this second `if` on the same line? O_o

Reply:
I guess not... fixing